### DISCUSSION TITLE
OIDCRedirectURI in a Reverse Proxy Setup

### DISCUSSION THREAD

**Discussion by bendercodes** [OpenIDC/mod_auth_openidc#722] [Q&A]
So when using this module in a reverse proxy setting, where the reverse proxy is proxying requests to different services using an Authorization grant, e.g:

```
OIDCClientID "DummyId"
OIDCCryptoPassphrase "DummyPass"
OIDCRedirectURI https://example.com/redirect/oidc
/... more unrelated settings

<Location "/serviceA">
  AuthType openid-connect
  Require valid-user
</Location>

<Location "/serviceB">
AuthType openid-connect
  Require valid-user
</Location>

<Location "/serviceC">
  /... completely different legacy auth not using this module
</Location>
```

1) Can we have a single generic redirect URI (`OIDCRedirectURI https://example.com/redirect/oidc`) or does it need to be at the directive level (e.g. `OIDCRedirectURI https://example.com/serviceA/redirect`)

2) Do we have to create a dedicated Location directive for the redirect URI or is it sufficient to just list it as part of the configuration, e.g.:

`<Location "/redirect/oidc"`></Location>

Thank you in advance and I'm happy to submit a PR updating the FAQs if it's helpful for anyone else.

**Comment by zandbelt** [comment]
as the docs mention, it needs is to be protected by mod_auth_openidc so it can be anything that lives under `AuthType openid-connect`
